coding_style/README.adoc

```diff
@@ -204,3 +249,7 @@ Rationale:: This will help with both editor highlighting as well as identifying
 Avoid duplicating the remote full path in the role directory, however, as that creates unnecessary depth in the file tree for the role.
 Grouping sets of similar files into a subdirectory of `templates` is allowable, but avoid unnecessary depth to the hierarchy.
 ====
+
+* Using agnostic modules like `package` only makes sense if the features required are very limited.
+In many cases, if the platform is different, the package name is also different so that using `package` doesn't help a lot.
```

Comment:
I'm not completely sure anymore (Gitlab diff veiw isn't great in this case) but to me it looks like this is indented diffently than the other multi-line list itrems in this file

Reply:
GitLab diff view on GitHub is indeed not great :smirk: Fixed anyway.